portafly/src/i18n/locales/en/accounts.yml

```diff
@@ -1,20 +1,107 @@
+invitations_tab: Invitations
+invitations_tab_description: Label for the tab that contains the list of all invitations, selection criteria and actions that can be applied on them.
+
+## Tab: Listing
+## Toolbar
+#-------# Bulk selector: available in shared.yml
+#-------# Delete button: available in shared.yml
+team_members_filter: Name
+team_members_filter_description: List containing the name criterion to filter team members
+team_members_typeahead: Filter by name #----PROPOSED
+team_members_typeahead_description: Default text to show when the user wants to filter by typing the text
+
+team_members_table:
+  aria_label: Team members table
+  aria_label_description: Aria label of the table of team members
+  empty_state: Currently, there are no team members. Add team members. #----PROPOSED
+  empty_state_description: Message to show when there are no team members to show, and prompt the user to add members
+  name_header: Name
+  name_header_description: Column listing names of team members
+  email_header: Email
+  email_header_description: Column listing the email addresses of team members
+  sso_authentication_heaer: SSO authentication
+  sso_authentication_header_description: Column listing if SSO authentication has been configured for team members
+  sso_authentication_value:
+    yes: Yes
+    not_yet: Not yet
+  sso_authentication_value_description: Container of possible values about users having SSO configured
+  role_header: Role
+  role_header_description: Column listing the roles associated to each team member
+  role_header_value: #----PROPOSED
+    admin_you: Admin (you)
+    admin: Admin
+    member: Member
+  role_header_value_description: Container with possible values of the roles of each team member
+  permission_groups_header: Permission groups
+  permission_groups_header_description: Column listing the permisions mapped to each team member
+
+## Tab: Invitations
+## Toolbar
+#-------# Bulk selector: available in shared.yml
+#-------# Delete button: available in shared.yml
+invitations_filter:  #----PROPOSED
+ by_recipient: Recipient
+ by_acceptance: Acceptance
+invitations_filter_description: List containing the criteria to filter invitations
+invitations_typeahead:
+  filter_by_recipient: Filter by recipient
+  filter_by_acceptance: Filter by acceptance
+invitations_typeahead_description: Default text to show when the user filters by recipient or acceptance
+
+invitations_table:
+  aria_label: Invitations table
+  aria_label_description: Aria label of the table of invitations
+  empty_state: Currently, there are no invitations sent. Start sending invitations to your team members. #----PROPOSED
+  empty_state_description: Message to show when there are no invitations to show, and prompt the user to send some invites
+  recipient_header: Recipient
+  recipient_header_description: Column listing the email addresses of invitation recipients
+  sent_header: Sent on UTC time #----PROPOSED
```

Comment:
@cntlsn I'm not sure, but it's possible to have other timezones, not only UTC ?

Reply:
@cntlsn Due to accessibility, special characters such as parentheses are discouraged. I'd simply place "Sent" , and explain in the docs that it uses UTC time. In this way, if other time zone systems are used, we update the docs instead of the UI. WDYT?

Reply:
@damianpm it is possible to select a different time zone, but user needs to go to settings and change it at the Account level (not at the personal level).
@porueesq for the reason I mentioned above I think it is hard for a user to understand exactly what the date is referred to (I'm thinking of an edge case where multiple members in different time zones share the same account). for this reason if we don't want to display the timezone in the table header I think we should at least display a help icon that would trigger a popover including:
* info about the current time zone
* info about how to change it (if user has permission)

Reply:
@porueesq I don't think adding the UTC info in the docs is enough, the user need to see that info in the UI. I'd keep it in the table by now. 

The other option @cntlsn mentioned (the help icon) could be a good idea, but I say let's keep it like this in the scope of this PR, and later we could add the help icon if needed (I think we need to do more research, for example to see if we are going to show in the table other timezones for an edge case)

Reply:
@cntlsn @damianpm Two columns might require the popover with info about the current time zone and info on how to change it: Sent and Accepted.
On one hand, adding a popover for each column implies duplication. I'd go with a message on top of the table that can be dismissed by the user. WDYT?